Enhancement/testimonials fadein scroll

jaideepkathiresan opened the pull request:
Fixes Issue #55 
Added a smooth fade-in scroll animation to the Testimonials section to match the rest of the site's design.

jaideepkathiresan commented:
Hi @Code-By-Adarsh, could you test this PR and let me know your feedback?

Code-By-Adarsh commented:
Why not bro? 
I will check and inform you in 15 minutes. 
I am going to college, so I am travelling now as I reach instantly inform you. 

Thank you @jaideepkathiresan your dedication.

Code-By-Adarsh commented:
Bro @jaideepkathiresan sorry but I cannot see animation can you calrify what have you changed? 

Please give me points about your changes. 

Then I will check again and inform you.

jaideepkathiresan commented:
Yeah, I saw that too, the animation time you had given i.e 0.6s to 0.8s is very minimal and not significan enough to see the changne, can I increase the time to fade-in?
changes i had made
1. 
Homepage CSS/testinomial.css

Lines 254–264: Added the .mk-testimonials-animate and .mk-testimonials-animate.visible classes to define the fade-in animation styles.
2. 
Homepage JS/testinomial.js

Lines 205–225: Added the IntersectionObserver logic to detect when the user scrolls to the Testimonials section and trigger the animation.
3. 
index.html

Line 479: Added the mk-testimonials-animate class to the Testimonials <section> tag.

Code-By-Adarsh commented:
Yes bro you can change anything as that matter. 
I am not developer but having a good knowledge about html, css and basic js.

So I understand what you said so please changes that make it cool, smooth, eye catching and as your creativity. 

Thanks bro @jaideepkathiresan

jaideepkathiresan commented:
Hi @Code-By-Adarsh, I have done the necessary changes now for the animation to be better visible. Could you test the latest commit?

Code-By-Adarsh commented:
Yes bro @jaideepkathiresan , I am currently now in my practical as I leave it I will check and inform you instantly. 

Thank you bro for your dedication for Mail Karo.

Code-By-Adarsh reviewed (approved):
Perfect work bro @jaideepkathiresan
